EXAMPLES: Combine WebGL Lines Colors and Splines

XanderLuciano opened the pull request:
#13439 - [Examples Cleanup](https://github.com/mrdoob/three.js/issues/13439#issuecomment-372059303)

> @mrdoob These two can be combined (Both in the same scene)
> https://threejs.org/examples/canvas_lines_colors.html
> https://threejs.org/examples/canvas_lines_colors_2d.html

This PR combines the two scenes of WebGL_lines_colors and WebGL_lines_splines into a single example.

Lines Colors had additional effects applied to the scene which were unnecessary so those were not merged.

Lines Spinles used `BufferGeometry` while Lines Colors used `Geometry`, however I have include both in this example as I think it makes for a better example to show both methods of adding color to `Three.Line`.

XanderLuciano committed:
create combination of webgl_lines_splines and webgl_lines_colors

XanderLuciano committed:
remove old examples

XanderLuciano committed:
update title with new name

WestLangley commented:
Ephemeral test link: https://rawgit.com/XanderLuciano/three.js/dev/examples/webgl_lines_colorsplines.html

Mugen87 commented on examples/webgl_lines_colorsplines.html:
```diff
@@ -0,0 +1,280 @@
+
+				var buffGeometry0 = new THREE.BufferGeometry(),
+                    buffGeometry1 = new THREE.BufferGeometry(),
+                    buffGeometry2 = new THREE.BufferGeometry();
```
Can you please check your formatting? Looks like your are mixing soft and hard tabs. Please use just hard tabs.

WestLangley commented:
>Lines Splines used BufferGeometry while Lines Colors used Geometry, however I have include both in this example as I think it makes for a better example

My intention was to remove `Geometry` from the examples in favor of `BufferGeometry`. The reason is that, at some point, `Geometry` will no longer be renderable. Encouraging users to use `BufferGeometry` now will make for an easier transition.

However, we should do whatever @mrdoob feels is best.

XanderLuciano committed:
replace spaces with tabs

XanderLuciano commented:
@WestLangley Not a problem at all, I left the splines example in the `BufferGeometry` section, so removing the `Geometry` portion is fairly straightforward and won't affect the rest of the code if mrdoob wants to have `BufferGeometry` only.

@Mugen87 My IDE was set to spaces, I switched it to tabs and updated the PR. Sorry about that!

mrdoob commented:
> @mrdoob Remove or keep `Geometry` in these examples?

I would vote moving to `BufferGeometry`. But no need to be done in this PR.

However, I think I would keep the filename `webgl_lines_colors.html` instead of `webgl_lines_colorspline.html`.

XanderLuciano committed:
renamed example file back to webgl_lines_colors

mrdoob commented:
Thanks!
